infra/sync/src/main/java/com/simprints/infra/sync/usecase/CleanupDeprecatedWorkersUseCase.kt

```diff
@@ -37,12 +37,8 @@ internal class CleanupDeprecatedWorkersUseCase @Inject constructor(
 
     private fun tagsForDeprecatedWorkers() = listOf(
         "TAG_PEOPLE_SYNC_WORKER_TYPE_DOWN_COUNTER", // 2023.1.0
-        "TAG_PEOPLE_SYNC_WORKER_TYPE_DOWNLOADER", // 2023.1.0
-        "TAG_PEOPLE_SYNC_WORKER_TYPE_END_SYNC_REPORTER", // 2023.1.0
-        "TAG_PEOPLE_SYNC_WORKER_TYPE_START_SYNC_REPORTER", // 2023.1.0
         "MASTER_SYNC_SCHEDULERS", // 2023.1.0
         "TAG_PEOPLE_SYNC_WORKER_TYPE_UP_COUNTER", // 2023.1.0
-        "TAG_PEOPLE_SYNC_WORKER_TYPE_UPLOADER", // 2023.1.0
```

Comment:
Why did we add these tags for cleanup in the first place? Were they used for deprecated workers and then reused for the new ones? If that's the case, don't we still want to cancel deprecated workers? If so, we should rename the tags for the current workers instead of removing old ones from cleanup.

Reply:
This is intended to stop old workers scheduled in previous versions of
SID to prevent crashes during migration to newer app versions.

Reply:
Yes, and with the removal of these tags aren't we regressing to the same situation?